### DISCUSSION TITLE
Welcome to FetchXMLBuilder Discussions!

### DISCUSSION THREAD

**Discussion by rappen** [rappen/FetchXMLBuilder#429] [General]
## 👋 Welcome to FetchXML Builder Discussions!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sasrsc** [comment]
Brand new to XRM Toolbox and Fetch XML. I am a cit dev, but I landed on this because I'm trying to get the fetch xml query to copy / paste into Power Automate. I am using Dataverse. I build my query. In FetchXMLBuilder I can add my 2 tables (using the relationship). It does return the correct results on the primary table M:1 (in the M side) but it doesn't return any columns with data for the "1" side (the parent), yet the filter works. When I use that XML within Power Automate it returns everything I need which is great. Why does the "2nd table" not show the output. Is there a simple obvious setting/answer here. Sorry. Trying to learn.

**Reply by BetimBeja** [reply]
@sasrsc did you select any attributes in the Linked entity?

**Comment by sasrsc** [comment]
I shut down XRM and restarted and reran the query and it worked. Weird. Now the attributes show up. Sorry to be so silly. Should have done that before I sent the note.

The items in red weren't listed on the output panel. Now they are.

I had clicked the execute multiple times. Moving on...

Very excited about this tool though...thank you.
[cid:b89f26d2-1504-4701-8c6c-5e1dc1678506]




Richard Clowes

Senior Director ● Travel Operations

Tel: + 1 919 531 7955 ● Cell: + 1 919 413 3084 ● ***@***.******@***.***>

Worldwide Headquarters ● Cary, NC



www.sas.com<http://www.sas.com/>

SAS® … THE POWER TO KNOW®



________________________________

**Comment by tristonarisawariotinto** [comment]
Has anyone experienced the issue with Bulk Data Updater if you navigate away from the Tab and it finishes you cant actually get back into the tab to click on the popup that triggers when the records are done?

**Reply by rappen** [reply]
Hi @tristonarisawariotinto 

I believe this is the same issue, and here is how to "solve" it: https://github.com/rappen/BulkDataUpdater/issues/225#issuecomment-2175391534
